### PR TITLE
Add event check

### PR DESCRIPTION
**Pull request by ChuckCox**
This PR is to add an eventing check to the experience test.

### CODE REVIEW

**Comment by gee4vee** [comment]
@ChuckCox This is good, but ideally we'd want to test when it's empty, start the ping source, check for events, and then clear the DB and the ping source.

File: cronjob-source.yml [hunk 1]
```diff
@@ -1,11 +0,0 @@
-apiVersion: sources.eventing.knative.dev/v1alpha1
```
Comment (gee4vee) [inline cronjob-source.yml:1]: Is there a reason why this file was deleted?

Comment (ChuckCox) [reply cronjob-source.yml:1]: We aren't using this file. It uses a deprecated 'kind' of "CronJobSource", which has been replaced by "PingSource", as of Knative v0.13.

Comment (ChuckCox) [reply cronjob-source.yml:1]: https://knative.dev/docs/eventing/migration/ping/

**Pull request closed without merging**